README.md

```diff
@@ -1 +1,226 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/">JavaScript</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://developer.mozilla.org/">CSS</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://developer.mozilla.org/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://developer.mozilla.org/">Python</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Framework</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://djangoproject.org/">Django Rest Framework</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Other Technologies</summary>
+  <ul>
+    <li><a href="https://npmjs.com/">NPM</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://pypi.com/">PIP</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="http://localhost:3000/">Express.js</a></li>
+  </ul>
+  <ul>
+    <li><a href="http://localhost:8000/">Django Developemnt Server</a></li>
+  </ul>
+</details>
+
+<details>
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **[Landing page]**
+- [ ] **[Register]**
+- [ ] **[Loging]**
+- [ ] **[Verify Email]**
+- [ ] **[Reset Password]**
+- [ ] **[Get all Debtors]**
+- [ ] **[Get single debtor]**
+- [ ] **[Debtor profile]**
+- [ ] **[Complaints/Suggestions]**
+- [ ] **[Simple_and_easy_to_use_UI]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+[]Frontend
+- Node 
+- npm
+
+[]Backend
+- PIP
+- Python v3.8
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-folder
+  https://github.com/zuri-training/Team-opossum-debtor.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Team-opossum-debtor
+  npm install
+
+  []For Backend
+  pip install -r requirements.txt
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+[] Frontend:
+```sh
+  npm run build
+  npm run start
+```
+[] Backend:
+```sh
+ python manage.py runserver
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Contributors <a name="authors"></a>
+
+👤 **Chukwuji Chinaza**
+
+- GitHub: [@Chukwuji](https://github.com/Chukwuji)
+- Twitter: [@Chukwuji](https://twitter.com/Chukwuji)
+- LinkedIn: [Chukwuji](https://linkedin.com/in/Chukwuji)
+
+👤 **Emmanuel Simasiku**
+
+- GitHub: [@Mukumbuta](https://github.com/Mukumbuta)
+- Twitter: [@Mukumbuta8](https://twitter.com/Mukumbuta8)
+- LinkedIn: [Mukumbuta](https://linkedin.com/in/mukumbuta)
+
+👤 **Chimezie Ochuba**
+
+- GitHub: [@chimezdev](https://github.com/chimezdev)
+- Twitter: [@chimezdev](https://twitter.com/chimezdev)
+- LinkedIn: [chimezdev](https://linkedin.com/in/chimezdev)
```

Comment:
- LinkedIn: [Chimezie Livingstone](https://www.linkedin.com/in/livingstone-chimezie)

Reply:
Hi @chimezdev 
Thank you for the review